include/tvm/ir/diagnostic_context.h

```diff
@@ -18,71 +18,25 @@
  */
 
 /*!
- * \file diagnostic.h
+ * \file diagnostic_context.h
```

Comment:
now that we have removed diagnostic.h I think it is better to just rename back to diagnostic.h for simplicity